solr/solrj/src/test/org/apache/solr/client/solrj/io/stream/eval/ArrayEvaluatorTest.java

```diff
@@ -54,12 +53,12 @@ public void arrayLongSortAscTest() throws IOException {
 
     result = evaluator.evaluate(new Tuple(values));
 
-    Assert.assertTrue(result instanceof List<?>);
+    assertTrue(result instanceof List<?>);
 
-    Assert.assertEquals(3, ((List<?>) result).size());
```

Comment:
can't it be casted once ?

Reply:
I'll take a look

Reply:
Fixed in 3e46998f859ac463dd68c75f35fe177fba1323a2